File: STL/stack.cpp
/*
----Stack-----
s.push(val)
s.pop()
s.top()
s.size()
s.empty()
*/

#include<iostream>
#include<stack>
using namespace std;
int main()
{
    stack<int>s;
    s.push(1);
    s.pop();
    if(s.empty())
    {
        cout<<"stack is empty"<<endl;
    }
    cout<<s.top()<<endl;
    return 0;
}

File: Generics/Stack.cs
using System;
using System.Collections.Generic;
public class Program
{
    public static void Main(string[] args)
    {
        Stack<int> stack = new();

        stack.Push(1);
        stack.Push(2);

        while(stack.Count != 0)
        {
            int top = stack.Peek();
            Console.Write($"{top}\t");
            stack.Pop();
        }
    }
}